test/UnitTest/Microsoft.AspNetCore.OData.Test/Microsoft.AspNetCore.OData.Test.csproj

```diff
@@ -33,13 +34,14 @@
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="2.0.1" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-    <PackageReference Include="Microsoft.OData.Core" Version="7.6.1" />
-    <PackageReference Include="Microsoft.OData.Edm" Version="7.6.1" />
-    <PackageReference Include="Microsoft.Spatial" Version="7.6.1" />
+    <PackageReference Include="Microsoft.OData.Core" Version="7.7.1" />
+    <PackageReference Include="Microsoft.OData.Edm" Version="7.7.1" />
+    <PackageReference Include="Microsoft.Spatial" Version="7.7.1" />
     <PackageReference Include="Moq" Version="4.7.137" />
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="System.Text.Json" Version="4.7.1" />
```

Comment:
@xuzhg Might [this issue](https://github.com/OData/odata.net/issues/1878) in odata.net have any bearing on the version of System.Text.Json here?

Reply:
When we fix [this issue](https://github.com/OData/odata.net/issues/1878), we won't this package reference at all. But, now, we have to include this package reference.